apps/ensnode/test/label-helpers.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, it } from "vitest";
+
+import { heal } from "../src/lib/label-helpers";
+
+describe("heal", () => {
+  it("heals a labelhash", async () => {
```

Comment:
```suggestion
  it("heals a known labelhash", async () => {
```

And then suggest creating a separate unit test for the case of a labelhash that is properly formatted but that is not known / not healable.